build.sh

```diff
@@ -170,60 +164,22 @@ function docker_build_with_version {
 
   pull_image "$dockerfile"
 
+  is_podman() {
+    local _bin
+    _bin=$(docker --version | cut -d" " -f1)
+    [[ $_bin == "podman" ]] && echo true
+  }
+
+  if [[ "$SKIP_SQUASH" == 0 ]] && is_podman; then
+    BUILD_OPTIONS+=" --squash"
+  fi
   # shellcheck disable=SC2016
   parse_output 'docker build '"$BUILD_OPTIONS"' -f "$dockerfile" "${DOCKER_BUILD_CONTEXT}"' \
                "tail -n 1 | awk '/Successfully built|(^--> )?(Using cache )?[a-fA-F0-9]+$/{print \$NF}'" \
                IMAGE_ID
-  clean_image
```

Comment:
Why remove the `clean_image` call over here?

Reply:
A mistake.

Reply:
I need to remove the `clean_image` call, because if we clean the built image we will not have an image to test on.